infra/terraform/cache.tf

```diff
@@ -29,6 +52,28 @@ module "secondary_cache" {
   log_analytics_workspace_id  = module.hub_app_insights[0].log_analytics_workspace_id
 }
 
+resource "azurerm_redis_cache_access_policy_assignment" "secondary_current_user" {
+  count              = var.environment == "prod" ? 1 : 0
+  name               = "secondarycurrentuser"
+  redis_cache_id     = module.secondary_cache[0].cache_id
+  access_policy_name = "Data Owner"
+  object_id          = data.azuread_client_config.current.object_id
+  object_id_alias    = "currentuser"
+}
+
+resource "azurerm_redis_cache_access_policy_assignment" "secondary_app_user" {
+  count              = var.environment == "prod" ? 1 : 0
+  name               = "secondaryappuser"
+  redis_cache_id     = module.secondary_cache[0].cache_id
+  access_policy_name = "Data Contributor"
+  object_id          = azurerm_user_assigned_identity.secondary_app_service_identity[0].principal_id
+  object_id_alias    = azurerm_user_assigned_identity.secondary_app_service_identity[0].principal_id
+
+  depends_on = [
+    azurerm_redis_cache_access_policy_assignment.secondary_current_user
```

Comment:
maybe leave a comment as an fyi that access policy assignments must be done serially, hence why you're depending on it

Reply:
It looks like this was fixed - https://github.com/hashicorp/terraform-provider-azurerm/pull/26085.  I'm going to get rid of this dependency.